store/store.go

```diff
@@ -1642,7 +1642,7 @@ func (s *Store) snapConnCheck() ([]string, error) {
 
 var ErrStoreOffline = errors.New("store is marked offline, use 'snap unset system store.access' to go online")
 
-func (s *Store) checkStoreOnline() error {
+func (s *Store) checkStoreOffline() error {
```

Comment:
this rename is strange, as I would expect such a function to error if the store is not offline, which is not what it does.

Reply:
Yeah, I was debating it. I'll revert that.